### DISCUSSION TITLE
Welcome to Debian_on_Buffalo Discussions!

### DISCUSSION THREAD

**Discussion by 1000001101000** [1000001101000/Debian_on_Buffalo#69] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by das-chameleon** [comment]
Hello 
the zip-download failed. please can you check or fix this?

**Reply by 1000001101000** [reply]
Are you referring to the releases section?

If so I just cleared those out, those are very outdated and likely wouldn't work. 

To get the latest (automatically updated) files you can download the desired ones individually or clone the repository.

Thanks!

**Comment by das-chameleon** [comment]
happy new year, sorry but i am not that familiar with github.  I've now added the repository in Github Desktop.  thank you for your reponse

Von meinem iPhone gesendet

**Reply by 1000001101000** [reply]
Excellent! Let me know if you run into problems.

**Reply by Lutz24146** [reply]
Thank you so much for all your hard work!
Unfortunately, I can't clone the repository. I have a profile on Github, and my username and password aren't accepted.
I wanted to try the debootstrap for Buffalo LS-WSXL.
Fatal: Authentication failed for 'https://github.com/1000001101101000/Debian_on_Buffalo/'
root@eeepc:/media/sdd1# git clone --depth 1 https://github.com/1000001101101000/Debian_on_Buffalo
Clone to 'Debian_on_Buffalo'...
Username for 'https://github.com': Lutz24146
Password for 'https://Lutz24146@github.com':
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
Fatal: Authentication failed for 'https://github.com/1000001101101000/Debian_on_Buffalo/'
Thank you very much!
[grosse.lutz@outlook.com]

**Reply by 1000001101000** [reply]
That's all coming from GitHub and includes a link with options for correcting the issue. Personally, I use ssh-keys when I clone repos but there are are other options as well.

**Comment by Strandvaskeren** [comment]
Thank you so much for putting in the effort to put debian on buffalo boxes.

I started out with a Buffalo TS4400R which was super easy, it's basically a regular pc in a 1U box, with vga output and all, so a usb stick with ubuntu server and I was up and running in 20 minutes. 

Then I got a TS-RXL, found your github page and went to work. I've tried all the options you have in your github, tried compiling my own stuff, found somewhere that mv78100 cpu's only works to debian 7, tried do a debian 7 build on the harddrive before mounting it in the TS-RXL, banging my head against the wall, getting nowhere for 4 evenings straight.

With the official firmware the TS-RXL worked fine, no issues other than I prefer debian. Couldn't telnet to the device, but took care of that by moving the hdd to another system and adding the needed installer files there. Installer worked fine too, ssh installer@172.16.10.166 which was the dhcp address the box got, both running the installer and when running the official firmware. After installing and rebooting I no longer had access to the box, so I assumed it was about the mv78100 cpu having issues with the pci bus and nics. I tried all sorts of tricks, went back to the official firmware and connected to 172.16.10.166 without issues, ran the installer again on 172.16.10.166 and every single time the installer was done there were no access to the ip.

Then tonight I tried the installer again for the xx'th time, installed fine, rebooted, then no contact to the box. Then I looked at my dhcp leases and saw that the mac address of the TS-RXL was now given the ip 172.16.10.175. Did ssh 172.16.10.175 and everything worked perfectly, debian up and running, thumbs up, no problems.

I have zero clue why my dhcp server hands the ip 172.16.10.166 to the mac address of the TS-RXL when it runs the official firmware and when it runs the installer image, but hands out 172.16.10.175 to the exact same mac address once the TS-RXL runs debian. Both ip addresses are listed with the same mac address in the dhcp leases. 

I tried restoring the official firmware and the TS-RXL went back to 172.16.10.166, same with the installer images running, but switched to 172.16.10.175 as soon as debian was installed.

It makes no sense to me, but at least it's up and running.

Thanks once more for putting in the time and effort cracking these buffalo boxes.

**Reply by 1000001101000** [reply]
Good to hear from you!

I get the same thing with my devices and Mikrotik router, or at least something similar. 

I believe older dhcp clients like those in the stock firmware and the debian installer just send the MAC address to the dhcp server. The newer version found it the past few Debian releases include an additional suffix. For some routers this is treated as a mismatch for dhcp. Usually re-creating the static address reservation or whatnot is sufficient to deal with it.

**Comment by Nash-254** [comment]
Hi all,

New Here. Looking to install a NAS OS on my Buffalo 421de and I came across this and OMV in my google searches. Read on the OMV forum someone suggecting to install Debian first then OMV ontop of it. Is that the process? Looking to give new life to the NAS; it's a really nice device and gave me service for quite a while and have been keeping it since to perhaps one day put it back to use after replacing it with a QNAP one.

**Reply by 1000001101000** [reply]
Last I knew OMV supported ARMHF and had instructions for installing to an existing Debian install so it should be possible.

**Comment by giupsspro** [comment]
Salve,
mi chiamo Giuseppe ed ho provato a seguire la guida per installare DEBIAN sul mio NAS TS3400D. Ho passato i file ma sono bloccato sul partizionamento dei dischi. Non sono una cima e quindi sicuro sbaglio qualcosa, ma dopo 15 gg di sbattimento mi trovo costretto a disturbarvi. La mia situazione è questa: ho questo NAS con 4 dischi da 4 tb. La mia domanda è: devo modificare solo le partizioni RAID1 e RAID0 (che sono le partizioni per come si vede nel video di Novaspirit tech) oppure tutte le partizioni sotto la voce Scas1, Scs2 ecc.? Grazie a chi dedicherà un pò del suo tempo ad aiutarmi!

**Comment by 1000001101000** [comment]
Good to hear from you!

I would think the partitioning would match the video except for the raid arrays spanning all 4 disks rather than just two. The most important part is that the "partition 1" on each disk is a suitable size ~1GB and that they are in a RAID1 array formatted ext3 and mounted on /boot. the rootfs and swap should be on separate RAID1 partitions as well  though the details aren't as important for those. The rest of the disks can be partitioned/raided how ever you like.
